projects/core/src/occ/adapters/site-context/default-occ-site-context-config.ts

```diff
@@ -10,11 +10,6 @@ export const defaultOccSiteContextConfig: OccConfig = {
         regions:
           'countries/${isoCode}/regions?fields=regions(name,isocode,isocodeShort)',
         baseSites:
-          'basesites?fields=DEFAULT,baseSites(defaultPreviewCatalogId,defaultPreviewCategoryCode,defaultPreviewProductCode,urlEncodingAttributes)',
```

Comment:
also need to replace `baseSitesForConfig` to `baseSites` in `occ-site.adapter.ts`.